testcases/OpTestEEH.py

```diff
@@ -379,3 +385,18 @@ def check_eeh_pe_recovery(self, pe):
             if pe in device:
                 return True
         return False
+
+
+    ##
+    # @brief  This function is used to get count of interfaces in PCI devices
+    #         available at that instant of time.
+    #
+    # @returns count @type int: interface count
+    #
+    def get_count_of_interdaces_in_pci_devices(self, pci_addr):
+        cmd = "ls -l /sys/class/*/ -1"
```

Comment:
yes, this will be fine or you can remove "wc -l", only corresponding PE is good to list
`cmd = "ls -l /sys/class/*/ -1 | grep %s " % pci_addr.`